.github/workflows/tests.yml

```diff
@@ -34,6 +34,27 @@ jobs:
         working-directory: microcloud
         run: make check-static
 
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.20.x
+
+      - name: Install dependencies
+        run: |
+          sudo add-apt-repository ppa:dqlite/dev -y
+          sudo apt-get install -y libdqlite-dev pkg-config
```

Comment:
We tend to add `--no-install-recommends` to `apt-get install` commands to avoid pulling in unnecessary packages.